<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelShowResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnStart">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>130</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始计算</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHelp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>450</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnBin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>二进制数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnDec">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>十进制数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnHex">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>16进制数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnSym">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>符号</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>487</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>选择(&amp;H)</string>
    </property>
    <addaction name="actionOrdinary"/>
    <addaction name="actionLog"/>
    <addaction name="actionPower"/>
    <addaction name="actionCompare"/>
    <addaction name="separator"/>
    <addaction name="actionTri"/>
    <addaction name="actionFirst"/>
    <addaction name="actionQuadratic"/>
    <addaction name="actionInver"/>
    <addaction name="actionExpLog"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOrdinary">
   <property name="text">
    <string>普通计算</string>
   </property>
   <property name="statusTip">
    <string>选择进行进制转换</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>十进制数_对数运算</string>
   </property>
   <property name="toolTip">
    <string>对数</string>
   </property>
   <property name="statusTip">
    <string>选择进行对数计算</string>
   </property>
  </action>
  <action name="actionPower">
   <property name="text">
    <string>十进制数_幂运算</string>
   </property>
   <property name="statusTip">
    <string>选择进行幂运算</string>
   </property>
  </action>
  <action name="actionCompare">
   <property name="text">
    <string>比较运算</string>
   </property>
   <property name="statusTip">
    <string>选择进行比较运算</string>
   </property>
  </action>
  <action name="actionTri">
   <property name="text">
    <string>三角函数</string>
   </property>
   <property name="statusTip">
    <string>选择进行三角函数计算</string>
   </property>
  </action>
  <action name="actionFirst">
   <property name="text">
    <string>一元一次函数</string>
   </property>
   <property name="statusTip">
    <string>选择进行一元一次函数计算</string>
   </property>
  </action>
  <action name="actionQuadratic">
   <property name="text">
    <string>一元二次函数</string>
   </property>
   <property name="statusTip">
    <string>选择进行一元二次函数计算</string>
   </property>
  </action>
  <action name="actionInver">
   <property name="text">
    <string>反比例函数</string>
   </property>
   <property name="statusTip">
    <string>选择进行反比例函数计算</string>
   </property>
  </action>
  <action name="actionExpLog">
   <property name="text">
    <string>指数对数函数</string>
   </property>
   <property name="statusTip">
    <string>选择进行指数对数函数计算</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
